README.md

```diff
@@ -18,16 +18,16 @@ Here is the [MeiliSearch documentation](https://docs.meilisearch.com/) 📖
 
 #### Quickstart
 ```java
-import meilisearch.Config;
-import meilisearch.Indexes;
-import meilisearch.Client;
+import meilisearch.MeilisearchConfig;
```

Comment:
Because they are all in `meilisearch` package, seems name becomes too verbose(just personal suggestion, so don't change immediately).
Is there some reason changing name?

Reply:
I am a true believer that class names must be really explicit and precise about what they do/are, that's why I changed it to this more verbose name.
Though I agree it might be too verbose and having it included in the `meilisearch` package is already a good indicator of what is does.
Let me know if you want it changed 👍

Reply:
I do agree with @djKooks, if you think this is the best option it is ok, but IMHO it is clear enough using `meilisearch.Client` instead of `meilisearch.MeiliSearchClient`

NOTE: By convention, we use the name `MeiliSearch` with a capital S as in a 2 word expression, unless it is of course against the language normal syntax ;)

Reply:
@eskombro thanks for suggestion.
@AmiralBl3ndic I made some research, and seems making name as before will be better(`meilisearch.client`) after comparing several open source projects(other meilisearch SDK, spring, apache projects...).
But if you have any other references, please share us so we can think about it.